host/sgx/create.c

```diff
@@ -589,6 +604,17 @@ oe_result_t oe_sgx_validate_enclave_properties(
         goto done;
     }
 
+    if (!(properties->config.attributes & OE_SGX_FLAGS_KSS))
+    {
+        if (!oe_sgx_is_unset_uuid(
+                (uint8_t*)properties->config.extended_product_id))
```

Comment:
If either `extended_product_id` or `family_id` is not all zeros, we have to return failure besides logging.

Reply:
Updated